html/controls/plt/plt-input.js

```diff
@@ -24,16 +24,18 @@ function preparePltInputTable(element, teamId, mode, statsbookPeriod, alternateN
 		var table = $('<table cellpadding="0" cellspacing="0" border="1">').addClass('PLT Team').addClass("AlternateName_" + alternateName).appendTo(element);
 		var thead = $('<tr>').appendTo($('<thead>').appendTo(table));
 		if (mode == 'plt' || mode == 'lt') {
-			$('<td>').text('Bench').appendTo(thead);
-			$('<td>').text('Jammer').appendTo(thead);
-			$('<td>').text('Pivot').appendTo(thead);
-			$('<td>').text('Blocker').appendTo(thead);
-			$('<td>').text('Box').appendTo(thead);
 			if (mode == 'lt') {
-				$('<td>').attr('id', 'head').text('Team ' + teamId).appendTo(thead);
+				$('<td>').attr('colspan', '5').attr('id', 'head').text('Team ' + teamId).appendTo(thead);
 			} else {
-				$('<td>').appendTo(thead);
+				$('<td>').text('Bench').appendTo(thead);
+				$('<td>').text('Jammer').appendTo(thead);
+				$('<td>').text('Pivot').appendTo(thead);
+				$('<td>').text('Blocker').appendTo(thead);
+				$('<td>').text('Box').appendTo(thead);
 			}
+			$('<td>').attr('id', 'StarPass').text('SP').click(function() {
+				WS.Set('ScoreBoard.Team('+teamId+').StarPass', !$(this).hasClass('Active'));
```

Comment:
Can it be made more visually obvious that this is a button?

Reply:
Do you have a suggestion as to how (especially given that none of the other buttons on this screen is specifically marked as such)?

Reply:
I was thinking either a different background, or putting round corners inside the td.

Reply:
I've gone for a blue background as no other button on the screen has round corners.